stix_shifter_modules/cbcloud/stix_transmission/results_connector.py

```diff
@@ -0,0 +1,36 @@
+import json
+
+from stix_shifter_utils.modules.base.stix_transmission.base_results_connector import BaseResultsConnector
+from stix_shifter_utils.utils.error_response import ErrorResponder
+from stix_shifter_utils.utils import logger
+
+
+class ResultsConnector(BaseResultsConnector):
+    def __init__(self, api_client):
+        self.api_client = api_client
+        self.logger = logger.set_logger(__name__)
+
+    def create_results_connection(self, search_id, offset, length):
+        # TODO: confirm offset starts at 0 or 1 (CbCloud starts at 0)
```

Comment:
I think it starts at 0. And length value I the number of results

Reply:
Comment removed as functioning has been clarified.